lib/content-services/src/lib/document-list/components/document-list.component.ts

```diff
@@ -58,7 +56,7 @@ import {
     AlfrescoApiService,
     UserPreferenceValues,
     DataRow,
-    DataTableService
+    DataTableService, DataTableSchema, DataColumn
```

Comment:
Could you please format it properly (every type in separate line like all above types)